src/Data/StrMap.js

```diff
@@ -6,7 +6,7 @@
 exports._copy = function (m) {
   var r = {};
   for (var k in m) {
-    if (m.hasOwnProperty(k)) {
+    if ({}.hasOwnProperty.call(m, k)) {
```

Comment:
Can we make a variable for `{}` and reuse it throughout please? `{}` does actually go through the process of constructing a new object every time, so if in heavily called code the overhead can be noticeable.